src/components/TreebankService/TreebankService.js

```diff
@@ -67,7 +67,6 @@ class TreebankService extends Component {
           responseFn(error(request, `Unsupported request: ${name}`, ResponseMessage.errorCodes.UNKNOWN_REQUEST));
```

Comment:
There are no changes to the `gotoSentence` section of the code because it works with `treebank-react` without any changes.